From 7b593644bebe63cf73233e835c18d7bbdb8d063f Mon Sep 17 00:00:00 2001
From: Mario Limonciello <mario.limonciello@dell.com>
Date: Thu, 19 Apr 2018 09:28:44 -0500
Subject: [PATCH] efi/fwupdate.c Correct a typographical error

gcc -Og -g3 -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -fpic -Werror -Wall -Wextra -fshort-wchar -fno-merge-constants -ffreestanding -fno-stack-protector -fno-stack-check --std=gnu11 -DCONFIG_x86_64 -I/usr/include/efi/ -I/usr/include/efi/x86_64/ -iquote/build/include "-DDEBUGDIR=L\"/\"" -mno-mmx -mno-sse -mno-red-zone -nostdinc -maccumulate-outgoing-args -DEFI_FUNCTION_WRAPPER -DGNU_EFI_USE_MS_ABI -I/usr/lib/gcc/x86_64-linux-gnu/7/include -c -o fwupdate.o fwupdate.c
fwupdate.c: In function 'msleep':
fwupdate.c:54:2: error: 'gBS' undeclared (first use in this function); did you mean 'BS'?
  gBS->Stall(msecs);
  ^~~
  BS
fwupdate.c:54:2: note: each undeclared identifier is reported only once for each function it appears in
/build/efi/Makefile:113: recipe for target 'fwupdate.o' failed
---
 efi/fwupdate.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/efi/fwupdate.c b/efi/fwupdate.c
index 464a226..35c9266 100644
--- a/efi/fwupdate.c
+++ b/efi/fwupdate.c
@@ -51,7 +51,7 @@ static int debugging;
 static VOID
 msleep(unsigned long msecs)
 {
-	gBS->Stall(msecs);
+	BS->Stall(msecs);
 }
 
 
